### PR TITLE
Linking to libraries which implement the design consideration

### PR DESCRIPTION
**Pull request by udidahan**

### CODE REVIEW

**Review by ckittel — APPROVED** [review]
LGTM

**Comment by ckittel** [comment]
#sign-off

File: docs/best-practices/auto-scaling.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ title: Autoscaling guidance
 titleSuffix: Best practices for cloud applications
 description: Review autoscaling guidance. Autoscaling is the process of dynamically allocating resources to match performance requirements.
 author: EdPrice-MSFT
-ms.date: 05/17/2017
+ms.date: 10/07/2022
```
Comment (ckittel) [inline docs/best-practices/auto-scaling.md:6]: We didn't do a maintenance review this article yet, and should still be in the queue for maintenance.  Please revert the date change.  Thanks.

```suggestion
ms.date: 05/17/2017
```

**Review by ckittel — CHANGES_REQUESTED** [review]

**Review by EdPrice-MSFT — APPROVED** [review]
Looks good

**Comment by EdPrice-MSFT** [comment]
#Sign-Off

**Review by Court72 — APPROVED** [review]

**Comment by Court72** [comment]
Hi @ckittel, @EdPrice-MSFT 

Merging is still blocked due to outstanding requested changes. Can you resolve the issue and `#sign-off` on the PR again?

Thanks

image

**Review by ckittel — APPROVED** [review]
LGTM

**Comment by udidahan** [comment]
Thanks everyone :)

**Pull request merged**